packages/webdriverio/src/test.ts

```diff
@@ -425,16 +714,28 @@ describe('@axe-core/webdriverio', () => {
         it('returns results when callback is provided', async () => {
           await client.url(`${addr}/index.html`);
           new AxeBuilder({ client }).analyze((err, results) => {
-            if (err) {
-              // Something _should_ happen with error
-            }
+            assert.isNull(err);
             assert.isNotNull(results);
             assert.isArray(results?.violations);
             assert.isArray(results?.incomplete);
             assert.isArray(results?.passes);
             assert.isArray(results?.inapplicable);
           });
         });
+
+        it('returns an error as the first argument', async () => {
+          await client.url(`${addr}/index.html`);
+          new AxeBuilder({ client, axeSource: 'throw new Error()' }).analyze(
+            (err, results) => {
+              try {
+                assert.isNull(results);
+                assert.isNotNull(err);
+              } catch (e) {
+                // Do something here ¯\_(ツ)_/¯
```

Comment:
You'll have to use promise change to make this work. You'll have to change this to be:

```suggestion
                done(e)
```

Otherwise this test can never fail.